src/style.css

```diff
@@ -1 +1,46 @@
 /* styles go here */
+
+.Info {
+  display: block;
+  padding: 0;
+  margin-top: 0;
+  margin-bottom: 10px;
+}
+
+.Registration {
+  display: block;
+  padding: 0;
+  margin-bottom: 10px;
+}
+
+.Form {
+  margin-top: 20px;
+}
+
+.Facts {
+  display: block;
+  padding: 0;
+  margin-bottom: 10px;
+}
+
+.Additional {
+  display: block;
+  padding: 0;
+  margin-bottom: 10px;
+}
+
+.Additional.task {
```

Comment:
rather, let's create some kind of general class that we will apply to all the necessary tags instead of these identical selectors

Reply:
Done!